src/frontend/magnify/src/types/maybe.type.ts

```diff
@@ -0,0 +1 @@
+export type Maybe<T> = T | undefined;
```

Comment:
You can group together `Maybe` and `Nullable` in `misc` file, you will probably have other type like these

Reply:
and IMO you can remove `type` in the file name, you are already in a type directory